docs/migrating-from-html5lib-sanitizer.rst

```diff
@@ -0,0 +1,70 @@
+.. highlight:: python
+
+=====================================
+Migrating from the html5lib sanitizer
+=====================================
+
+The `html5lib <https://github.com/html5lib/html5lib-python>`_ module `deprecated <https://github.com/html5lib/html5lib-python/blob/master/CHANGES.rst#11>`_ its own sanitizer in version 1.1. The maintainers "recommend users migrate to Bleach." This tracks the issues encountered in the migration.
+
+Migration path
+==============
+If you upgrade to html5lib 1.1+, you will start getting deprecation warnings when using its sanitizer. If you decide to follow the recommendation and use Bleach for sanitization, you should understand that Bleach has its own sanitizer (despite the fact that it vendors html5lib). You may therefore encounter slightly different results. Avoid Bleach 3.2.0 because it leaks the deprecation warnings (for the sanitizer in the vendored html5lib v1.1); this is fixed in 3.2.1+.
```

Comment:
This paragraph (and others) should get word wrapped. Among other things, it's easier for me to comment on individual lines in pull requests.

The two sanitizers are very different and have very different goals. The Bleach sanitizer is not a drop-in replacement for the html5lib one. It's almost guaranteed that you will have different results and have to tune the Bleach sanitizer to your needs.

People should always use the latest version of Bleach, so we can remove this last sentence.

```suggestion
If you upgrade to html5lib 1.1+, you may get deprecation warnings when using its sanitizer. If you
follow the recommendation and switch to Bleach for sanitization, you'll need to spend time tuning the
Bleach sanitizer to your needs because the Bleach sanitizer has different goals and is not a drop-in
replacement for the html5lib one.
```